recipes/inspector/build.sh

```diff
@@ -5,3 +5,5 @@ mkdir -p $PREFIX/bin
 
 cp *.py $PREFIX/bin
 chmod +x *.py
+
+exit 0
```

Comment:
This is not needed.

Reply:
Thanks, Martin. Noticed a "bash exit 1" error so I made this change. I will keep it as is if no potential issues.

Reply:
OK, but it is useless. 
The `exit 1` must have been due to some error earlier

Reply:
Thanks, Martin. I have removed this line.